### PR TITLE
Single product page

### PR DESCRIPTION
**Pull request by vidhi-thakur**
added a single project page
image

### CODE REVIEW

File: src/pages/singleProduct/SingleProduct.css [hunk 1]
```diff
@@ -0,0 +1,67 @@
+.singleProduct {
+  width: 100%;
+  max-width: 56.25rem;
```
Comment (JatuFaique) [inline src/pages/singleProduct/SingleProduct.css:2]: You can remove the width and just go with max-width to limit its width as per your value

**Review by JatuFaique — APPROVED** [review]
The code looks clean and the Implementation is clear.
Looks good to merge.

File: src/pages/singleProduct/SingleProduct.css (hunk 1 above)
Comment (vidhi-thakur) [reply src/pages/singleProduct/SingleProduct.css:2]: updated

File: src/pages/singleProduct/SingleProduct.js [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+            try {
+                if (productID) {
+                    const response = await fetchSingleProduct(productID)
+                    setSingleProduct(response.product)
+                }
+            } catch (e) {
+                console.error(e)
+            }
+        })()
+    }, [])
```
Comment (msdeshmukh009) [inline src/pages/singleProduct/SingleProduct.js:25]: You can also get the particular product by filtering over the existing products array.

**Review by msdeshmukh009 — APPROVED** [review]
The code looks fine. There are no products visible in the deploy preview so you might want to check that out.

**Pull request merged**